### ISSUE TITLE
Oops Reminder Content

### ISSUE THREAD

**Issue by sydneywalcoff** [hackforla/expunge-assist#1154]
### Overview
If you use browser navigation like (refresh, forward and backward) while in the letter generator, the app forgets all your previous answers. And because it uses your previous answers to determine the flow and eventually to form the letter, the app breaks and shows a blank white page.

We should:
a) Warn users not to use browser navigation like (refresh, forward and backward) while in the letter generator before they start #1152 
b) If a user does refresh, we should show a little Oops reminder (similar to the https://github.com/hackforla/expunge-assist/issues/1136, but smaller) reminding the user not to use browser navigation

### Action Items
- [x] UXR support for Content 
- [ ] Create content for a little doodle or something telling the user not to use browser navigation
- [ ] Add doodle content to #1141
   - [ ] Remove dependency label
   - [ ] Move to 'new issue approval' column

### Resources/Notes
- User flow journey on [Figma](https://www.figma.com/file/SRsM0BvWhtmFwpg23bKTLB/EA-Collaboration-(Design%2FContent)?type=whiteboard&node-id=2592-9103&t=ZYn2CjFupgXFhTwf-4)
- Sitemap on [Figma](https://www.figma.com/file/hYqRxmBVtJbDv9DJXV6nra/Expunge-Assist-Main-Figma?type=design&node-id=2982-24928&mode=design&t=c8fnUyJlJeg5Ug2N-0)

**Comment by cjankowski23** [comment]
Update
**Progress:** First set of content explorations have been written and will be shared with content team for feedback. 
**Estimated Time to Completion (ETC):** Approximately one week
**Blockers:** Team feedback to ensure the message fits voice & tone as well as purpose.

**Comment by cjankowski23** [comment]
Progress: First set of content explorations have been shared with content team. 
Estimated Time to Completion (ETC): Approximately one week
Blockers: Currently revising content and developing strategies to enhance error notifications overall, to ensure a consistent tone and voice.

**Comment by cjankowski23** [comment]
**Progress:** Met with Ellynna from the Design Team to discuss possible template ideas for the warning strategy.
**Estimated Time to Completion (ETC):** Presenting iterations to group on Friday all teams meeting.
**Blockers:** Contingent on feedback.

**Comment by cjankowski23** [comment]
**Progress:** Created and shared with Ellynna
**Estimated Time to Completion (ETC):** Completed
**Blockers:** None

**Comment by cjankowski23** [comment]
**Progress:** Iterations in progress
**Blockers:** Spoke with Sydney, "Return Home" means returning to the landing page. Discuss with team if we would like for users to return home (landing page) or the beginning of the letter generator from the oops page.

Double check if we would like to take the opportunity to use the new language "declaration letter" or use the previous terms for now, until the next update.

**Comment by cjankowski23** [comment]
**Progress** Shared user action/ flow questions regarding call to action at the bottom of oops page with the Design and Content team. 

Utilize [Site Map] as a spring board to creating user flows for potential user actions. (https://www.figma.com/file/hYqRxmBVtJbDv9DJXV6nra/Expunge-Assist-Main-Figma?type=design&node-id=2982-24928&mode=design&t=c8fnUyJlJeg5Ug2N-0)

**Blockers** Create and present different user flow diagrams and provide the content that would lead to each action for returning home, returning to onboarding, returning to the start of the letter generator, or providing two button options, for returning home or the start of the letter generator.

**Estimated Time to Completion ETC** Present ideas next Friday

**Comment by sylvia-nam** [comment]
Added user flow journey to resources, added task item on UXR support for Content. I will work with Claudia with issue updated once we chat, map out research support request.

**Comment by cjankowski23** [comment]
Time for an update! Please comment the following update:
**Progress:** Content has been updated and brought up at standup. 
**Estimated Time to Completion (ETC):** 1 week
**Blockers:** Verifying next steps

**Comment by SamHyler** [comment]
@sydneywalcoff asked a question about this issue, @cjankowski23 is going to have a new look in regards to those questions

**Comment by cjankowski23** [comment]
Feedback received from @sydneywalcoff 
Re: Oops Reminder.
I was looking at the Figma before prioritizing it and I realized that it’s not completely accurate.
(this one is less of a big deal) We’re not clearing the information because of privacy reasons. This is a side effect from not saying any data in the first place and it just happens to conveniently fall in line with our values regarding privacy.

The bigger deal being that it only references refreshing when this problem can be caused by any browser navigation (refreshing, and using the browser forward and back buttons). I’m thinking this might confuse people who use the browser navigation vs the app navigation and end up in the same place.  I think the content should reference all the ways you can do this and should instead encourage the user to use the app navigation (the purple forward/back buttons that we added)

***progress***
Shared new iterations during standup.
Somethings we've discussed as a team:
Concerns on stating purple forward back buttons are related to color blindness and accessibility. 
Maintaining a supportive, straightforward, trustworthy, mentor like voice.

***Estimated Time to Completion (ETC):*** 
Will share new iterations during standup next week.

***Blockers***
none

**Comment by cjankowski23** [comment]
Time for an update! Please comment the following update:
**Progress:** Completed, updated content on design.

**The page refreshed**
Your letter was erased.
 
Only use the forward and back buttons for navigation and to prevent losing your work.

**Estimated Time to Completion (ETC):** 1 week
**Blockers:** Ask design to check spacing is correct, if correct hand over to dev.

Thanks again!

**Comment by SamHyler** [comment]
Final designs were never updated in the master pages, I just did that now.
